crates/tests-e2e/examples/fund.rs

```diff
@@ -298,7 +298,7 @@ async fn open_channel(
     )
     .await?;
 
-    bitcoind.mine(10).await?;
+    bitcoind.mine(20).await?;
```

Comment:
I don't think it makes a difference. It's probably just random that it worked for you with 20 conf instead of 10 confs

Reply:
It's not *random*. I noticed that the channel was active when I was re-running the script, but never on the first run.
After trying it a few times, I realised that this was due to the amount of blocks mined.

Increasing this number lead to consistent improvement (at least in the channel being in `active` state).